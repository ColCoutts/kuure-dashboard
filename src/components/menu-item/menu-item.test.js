import React from 'react';
import { shallow } from 'enzyme';
import MenuItem from './menu-item.component';

describe('menuitem component', () => {
  it('renders a snapshot', () => {
    const wrapper =
    shallow(<MenuItem
      title="test"
      description="nother test"
      imageUrl="/image"
      history=""
      match=""
      linkUrl="testUrl"
      />);
    expect(wrapper).toMatchSnapshot();
  });
});
