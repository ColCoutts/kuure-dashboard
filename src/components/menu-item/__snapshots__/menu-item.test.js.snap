// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`menuitem component renders a snapshot 1`] = `ShallowWrapper {}`;
